project/iterative_sorting.py

```diff
@@ -5,15 +5,15 @@ def selection_sort( arr ):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        for j in range(i, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
         # TO-DO: swap
-
-
-
+        arr[i], arr[smallest_index] = arr[smallest_index], arr[i]
```

Comment:
nice. this also work in python to swap
a, b = b, a
or
arr[i], arr[i+1] = arr[i+1], arr[i]